packages/omg-js-rootchain/src/rootchain.js

```diff
@@ -148,7 +148,7 @@ class RootChain {
         [
           outputId.toString(),
           outputTx,
-          [],
+          '0x',
```

Comment:
how did this work before then 🤔does [] default to empty byte?

Reply:
seems like both of it works, but i'm just following the empty type from the abi